swarm/network/stream/stream.go

```diff
@@ -228,6 +235,17 @@ func NewRegistry(localID enode.ID, delivery *Delivery, syncChunkStore storage.Sy
 	return streamer
 }
 
+//we need to construct a spec instance per node instance
+func (r *Registry) setupSpec() {
```

Comment:
can we please have this as `setupProtocolSpec`? and the corresponding `createSpec` as `createProtocolSpec`? these function names are over-generalised

Reply:
Any specific reason for that? I have had many "obvious" wording in previous PRs required to be edited away, The variable was called `Spec` before and in `p2p/protocols/protocol.go` it is still just called `Spec`. So I don't see a over-generalization here?